docs/source/devguides/walkthroughs/DrivenMatlabAgent-Walkthrough.rst

```diff
@@ -116,7 +119,7 @@ config file of DrivenMatlabAgent, DrivenMatlabAgent will send it to Matlab
 application and receive commands to send to devices.
 
 Resources
-=========
+--------------
```

Comment:
Proper RST only has the length of the title/section.  This will cause less warnings during the running of make html.